lib/index.js

```diff
@@ -1,9 +1,12 @@
-import { log } from './utils/console-utils'
+import {parser} from './log-parser'
+import {builder} from './log-tree-builder'
+import LogEntry from 'log-entry'
 
+let head = new LogEntry('00:00:00', 'HEAD of the Log', 0) //TODO: parse date from Log
 
-log.json({
-  hello: 'world'
-})
-
+parser(
```

Comment:
this is confusing name, a function should be a verb, it should be either `logParser.parse` or just `parseLog`